include/grpc/census.h

```diff
@@ -79,6 +79,14 @@ int census_enabled(void);
 */
 typedef struct census_context census_context;
 
+/**
+ Returns the census_context of the current thread, creating one if necessary.
+ The object pointed to by the returned pointer is thread local, and thus may
+ be read without concurrency concerns, but should not be shared with other
+ threads.
+*/
+census_context* census_context_current();
```

Comment:
My understanding is that gRPC wants to avoid having per-thread contexts (and we can't rely on the threads being used for (e.g.) filters being the ones used by an RPC), so we have to always reference the context used in the relevant call structure. So not sure we can do this.